activemodel/CHANGELOG.md

```diff
@@ -123,11 +123,13 @@
 
     *Wojciech Wnętrzak*
 
-*   Change validates_acceptance_of to accept true by default.
+*   Change `validates_acceptance_of` to accept `true` by default besides `"1"`.
 
-    The default for validates_acceptance_of is now "1" and true.
-    In the past, only "1" was the default and you were required to add
-    accept: true.
+    The default for `validates_acceptance_of` is now `"1"` and `true`.
+    In the past, only `"1"` was the default and you were required to pass
```

Comment:
`In the past` implies a change that existed some time ago, and not just the one we are just changing.
I think it's used only at one place(change) that's just below in the changelog. Everywhere else we use `previously`.